example/app.jsx

```diff
@@ -11,13 +11,14 @@ import Glyphicon from 'react-bootstrap/lib/Glyphicon';
 import FormGroup from 'react-bootstrap/lib/FormGroup';
 import ControlLabel from 'react-bootstrap/lib/ControlLabel';
 import HelpBlock from 'react-bootstrap/lib/HelpBlock';
+import createReactClass from 'create-react-class';
 
 const spanishDayLabels = ['Dom', 'Lu', 'Ma', 'Mx', 'Ju', 'Vi', 'Sab'];
 const spanishMonthLabels = ['Enero', 'Febrero', 'Marzo', 'Abril', 'Mayo', 'Junio', 'Julio', 'Agosto', 'Septiembre', 'Octubre', 'Noviembre', 'Diciembre'];
 const wapperDivStyle = { border: '1px solid #ccc' };
 const scrollingDivStyle = { padding: '10px', height: '70px', overflow: 'auto' };
 
-const App = React.createClass({
+const App = createReactClass({
```

Comment:
As far as I understand, `React.createClass` can be replaced with `React.Component` and then there is no need to have a dependency on a `create-react-class` library. 
That's how another library did the same fix: https://github.com/Hacker0x01/react-datepicker/pull/822/files#diff-04c6e90faac2675aa89e2176d2eec7d8L34